docs/src/supported-platforms.md

```diff
@@ -0,0 +1,22 @@
+# thin-edge.io platform support
+
+This doc`s target is to have an point to come to and find out if thin-edge.io will work on your device.
```

Comment:
I don't think this sentence is helpful. I would simply remove it.

Reply:
Also, here you use  `  instead of  '